schema_editor/app/scripts/views/usermgmt/add-controller.js

```diff
@@ -0,0 +1,34 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function UserAddController($log, $state, ASEConfig, AuthService, UserService, Notifications, Utils) {
+        var ctl = this;
+        ctl.user = {};
+        ctl.userGroup = '';
+        ctl.groups = ASEConfig.api.groups;
+
+        ctl.submitForm = function() {
+
+            ctl.user.groups = [ctl.userGroup];
+
+            UserService.User.create(ctl.user, function() {
+                Notifications.show({text: 'User ' + ctl.user.username + ' created successfully',
+                                    displayClass: 'alert-info',
```

Comment:
I think the rest of the schema editor uses `alert-success` for create / save operations, so this should be switched to match.